lib/publiccloud/instance.pm

```diff
@@ -868,4 +868,11 @@ sub do_systemd_analyze_time {
     return @ret;
 }
 
+sub upload_supportconfig_log {
+    my ($self, %args) = @_;
+    $self->ssh_script_run(cmd => 'sudo supportconfig -R /var/tmp -B supportconfig -x AUDIT', timeout => 7200);
+    $self->ssh_script_run(cmd => 'sudo chmod 755 /var/tmp/scc_supportconfig.txz', timeout => 3600);
+    $self->upload_log('/var/tmp/scc_supportconfig.txz', failok => 1);
```

Comment:
Can this work? The supportconfig is still on the test instance, no?

Reply:
Also the VRs do not have the `scc_supportconfig.txz` file :thinking:

Reply:
look at previous VRs. And yes, that is PC subroutine.

Reply:
look at previous VRs. And yes, that is PC subroutine.